neqo-transport/src/connection/tests/datagram.rs

```diff
@@ -43,12 +45,12 @@ fn datagram_disabled_both() {
     assert_eq!(client.max_datagram_size(), Err(Error::NotAvailable));
     assert_eq!(server.max_datagram_size(), Err(Error::NotAvailable));
     assert_eq!(
-        client.add_datagram(DATA_SMALLER_THAN_MTU),
+        client.add_datagram(DATA_SMALLER_THAN_MTU, Some(1)),
         Err(Error::TooMuchData)
     );
     assert_eq!(server.stats().frame_tx.datagram, 0);
     assert_eq!(
-        server.add_datagram(DATA_SMALLER_THAN_MTU),
+        server.add_datagram(DATA_SMALLER_THAN_MTU, Some(1)),
```

Comment:
If you are using the same ID, then there is no reason to use the ID at all, right?